cmd/kusk/internal/kuskui/ui.go

```diff
@@ -0,0 +1,28 @@
+package kuskui
```

Comment:
@kylehodgetts I added this to replace testkube's UI library as we have a little bit different color scheme. Plus are not going to use tables and other elements. Can you take a look at this?